std/stdio.d

```diff
@@ -832,21 +832,19 @@ $(D rawRead) always reads in binary mode on Windows.
         return buffer;
     }
 
+    ///
     unittest
     {
-        static import std.file;
+        import std.file;
 
         auto deleteme = testFilename();
```

Comment:
If we're making this unittest part of the public docs, we really need to make the identifiers something better than `deleteme`. That just looks bad, as part of the official docs.

Also, this unittest uses `testFilename`, which AFAIK is a Phobos internal test function that users really shouldn't know about, and besides, how are they supposed to know what it does, what it returns, and what kind of side-effects it might have?

IMO such code examples that require interaction with the OS are better left as code snippets in the docs, especially when the associated unittests use Phobos internal functions.